Kernel/FileSystem/Inode.cpp

```diff
@@ -48,7 +48,7 @@ void Inode::sync()
 {
     if (is_metadata_dirty())
         (void)flush_metadata();
-    fs().flush_writes();
+    MUST(fs().flush_writes());
```

Comment:
This will panic the kernel if there is an error flushing writes. We should probably do the same thing as with the metadata flushing and ignore the error... and figure out how to propagate it higher than this function in a later PR.